client/me/purchases/vat-info/index.tsx

```diff
@@ -84,11 +100,58 @@ export default function VatInfoPage() {
 					</CardHeading>
 					<p className="vat-info__sidebar-paragraph">
 						{ translate(
-							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST"). */
-							"We currently only provide %(taxName)s invoices to users who are properly registered. %(taxName)s information saved on this page will be applied to all of your account's receipts.",
+							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST") or a generic fallback string of tax names */
+							'The %(taxName)s details saved on this page will be applied to all receipts in your account.',
 							{
-								textOnly: true,
-								args: { taxName: taxName ?? translate( 'VAT', { textOnly: true } ) },
+								args: { taxName: taxName ?? translate( 'tax (VAT/GST/CT)', { textOnly: true } ) },
+							}
+						) }
+						<br />
+						<br />
+						{ translate(
+							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST") or a generic fallback string of tax names */
+							/* translators: This is a list of tax-related reasons a customer might need to contact support */
+							'If you:' +
+								'{{ul}}' +
+								'{{li}}Need to update existing Tax ID details{{/li}}' +
+								'{{li}}Have been charged VAT as a VAT-Registered Business subject to reverse charges{{/li}}' +
```

Comment:
Should this use `%(taxName)s` also? I think this could actually be re-worded to avoid mentioning VAT at all if you write something like `if you have been charged taxes as a business subject to reverse charges`.

Reply:
I thought so at one point, too. 

However, it seems that the `Reverse chargeback` is only a thing in the EU. This is something specific that is separate from refunding tax that was charged to an exempt organization. 

Maybe we don't need this anymore, since all of our receipts have our vendor information on them. I couldn't quite tell from the description I found: https://www.revenue.ie/en/vat/what-is-vat/reverse-charge-self-accounting.aspx

Reply:
Another thought might be to only show it is the user is in the EU? Not sure if that makes it overly complicated.

Reply:
That would not be easy to do, I think. Alright, if this is what's most accurate, then I'm ok with it.

Reply:
Got some more feedback from the tax team here: p1688673851881119-slack-payments-tax-engine.

Will update to match the suggested `if you have been charged taxes as a business subject to reverse charges`

Reply:
Fixed.